admin/database/postgres/migrations/0005.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE orgs ADD COLUMN quota_projects INTEGER DEFAULT 5;
+ALTER TABLE orgs ADD COLUMN quota_deployments INTEGER DEFAULT 10;
+ALTER TABLE orgs ADD COLUMN quota_slots_total INTEGER DEFAULT 20;
+ALTER TABLE orgs ADD COLUMN quota_slots_per_deployment INTEGER DEFAULT 5;
+ALTER TABLE orgs ADD COLUMN quota_outstanding_invitations INTEGER DEFAULT 200;
+
+ALTER TABLE users ADD COLUMN quota_singleuser_orgs INTEGER DEFAULT 3;
```

Comment:
If application level defaults are non zero then what is the use of setting DB defaults to -1 when it will be used ? I mean when will I know the -1 is set by a user and not a DB default.

Reply:
I meant that when using `ADD COLUMN` and `NOT NULL`, you have to specify a `DEFAULT` clause. But this is not the place to configure defaults, so just put a fixed value for all of them here.

Reply:
Can also just have default 0 here. I just suggested `-1` as a way of setting infinite, instead of setting a high number – but we can just keep the checks simple, we can anyway just set a very high number if needed.